blog/src/app/blog/dojo-talks-lose-mind/page.tsx

```diff
@@ -0,0 +1,173 @@
+import { Container, Link, Stack, Typography } from '@mui/material';
+import { Metadata } from 'next';
+import Image from 'next/image';
+import { ReactNode } from 'react';
+import sampleProblemImage from './losemind.webp';
+
+export const metadata: Metadata = {
+    title: 'Do World Champions Lose Their Minds? | Dojo Talks',
+    description:
+        'DojoTalks',
```

Comment:
Can we get a more fledged out description? This is what appears in the search results on google